src/applications/personalization/profile-2/msw-mocks.js

```diff
@@ -97,10 +97,10 @@ const createTransactionRequestSuccessBody = {
 // couple of places but for testing purposes I'm reusing this body for _all_
 // transaction creation requests.
 const createTransactionRequestFailedError = {
-  title: 'Check Email Domain',
-  code: 'VET360_EMAIL304',
-  detail: 'AlphaNumeric ToplevelDomainName must be <= 63 Characters.',
+  code: 'VET360_EMAIL206',
+  detail: 'Confirmation Date can not be greater than sourceDate',
```

Comment:
I needed to change this to a code other than VET360_EMAIL304, otherwise the unit tests dependent on this error response would fail since the AlertBox copy would return `It looks like the email you entered isn’t valid. Please enter your email address again.` 😄

Reply:
oh crap! good catch. was this a major source of your issues?

Reply:
It was initially, but I caught this one pretty early on 😄  Just a big coincidence we were using this exact code in the mocks.